### ISSUE TITLE
don;t see update 1.20

### ISSUE THREAD

**Issue by sniperxnl** [zigbee2mqtt/hassio-zigbee2mqtt#154]
![image](https://user-images.githubusercontent.com/16321561/126533306-6415a640-ac25-4a22-b0c9-9dea45b706e1.png)
Updated today to 1.18, but dont see 1.20, why not?

**Comment by benogen** [comment]
The repo changed, are you sure you have the most recent one added?

**Comment by YvoTuk** [comment]
I've got the same issue.
Was looking to resolve an issue with an unsupported device due to a changed vendor name, found it was fixed about a month ago but couldn't get it to work.
When I checked the MQTT version I realized mine is still 1.18.1-1 where 1.20 was already released.

Whatever I try (auto update on/off), reloading on the add on store page, it doesn't come up with the latest version.

Not sure about the comment from benogen? Normally it would auto update?

**Comment by benogen** [comment]
Have you got the repo "https://github.com/zigbee2mqtt/hassio-zigbee2mqtt" or do you use the old one? Then it cannot update.

**Comment by YvoTuk** [comment]
ok, I was not sure, so I just added the https://github.com/zigbee2mqtt/hassio-zigbee2mqtt line to HA.
I'm now seeing two identical lines:
![MQTT](https://user-images.githubusercontent.com/78449057/126776803-1b56b5c5-7e6e-405b-9cb9-24449120225c.png)

Does that mean I had an old one? 
If so... what would be the best route to move to the new one? I don't want to destroy all stuff I've installed so far... that's a bit scary?

**Comment by benogen** [comment]
yeah probably. I removed the old line from the repository window. I think I then uninstalled and reinstalled the newer one but I am not sure. As long as you keep the "data_path" in line with the old installation, you should not lose anything. In the end, just make a backup first.

**Comment by Koenkk** [comment]
@ciotlosm I'm noticing many users do not see the update, could you check this?

**Comment by ciotlosm** [comment]
I was away. I will look at this today

**Comment by Tche333** [comment]
@YvoTuk I ended up with same issue as you do cause I did not notice the repository changes for this addon. A change of repository means HA is considering the new one as a totally different AddOn application. 
It turned out that I followed the same procedure as [https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/issues/134#issuecomment-855363394](davidstosik in issue 134) and now my Zigbee2MQTT AddOn is back online (the new one).
In order not to be confused by the 2 AddOn co-existing with same name, I use a slightly different approach:
(1) just copy/backup your existing Zgbee2MQTT AddOn configuration at first
(2) delete the repository of your existing 1.18 AddOn (pointing to danielwelch one)
  The AddOn is still then running (as seen in your Dashboard) and in my case there is no Zigbee2MQTT icon in front of it. 
(3) Stop the existing AddOn, remove it from side bar if it was
[ continue as davidstosik tested approach ] 
(4) install new repository and install correspondig AddOn
(5) configure it by pasting back your previous AddOn config
  Z2M AddOn would then use all files left by previous AddOn, inc the devices.
(6) start it, everything should be now fine on latest (1.20) release

Last step - not tested yet 
(7) delete the old 1.18 AddOn to leave a clean/migrated solution in place.

That way I did no mess up between new and old because at first they look exactly the same.

@Koenkk I did not notice - from HA - that existing AddOn was deprecated. I assume reported problem in this issue are related to my same experience - I had to go to its GitHub page to see it (after wondering why my AddOn was not updated automatically anymore). Then I figured out, trying out the update procedure.
Maybe we could bump a new version on the deprecated AddOn with an updated description so that 
(1) the user can see Z2M was updated with a (1.18-DEPRECATED) release tag
(2) the AddOn updated readme would point to the migration procedure.

**Comment by sanderdekroon** [comment]
It might be good to mention that -at least in my case- the location of the config files has changed. Previous releases used the `/share/zigbee2mqtt` folder, while the 1.20 release uses the `/config/zigbee2mqtt/` folder. Copying over the folder contents made the newer addon work immediately and retains the previous configured devices.

**Comment by benogen** [comment]
> It might be good to mention that -at least in my case- the location of the config files has changed. Previous releases used the `/share/zigbee2mqtt/` folder, while the 1.20 release uses the `/config/zigbee2mqtt/` folder.
> 
> Copying over the folder contents made the newer addon work immediately and retains the previous configured devices.

You could also use the config to use the old location, that works as well.

**Comment by dirkkohnen** [comment]
> @YvoTuk I ended up [with same issue as you do](https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/issues/154#issuecomment-885581318) cause I did not notice the repository changes for this addon. A change of repository means HA is considering the new one as a totally different AddOn application.
> It turned out that I followed the same procedure as [davidstosik in issue 134](https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/issues/134#issuecomment-855363394) and now my Zigbee2MQTT AddOn is back online (the new one).
> In order not to be confused by the 2 AddOn co-existing with same name, I use a slightly different approach:
> (1) just copy/backup your existing Zgbee2MQTT AddOn configuration at first
> (2) delete the repository of your existing 1.18 AddOn (pointing to danielwelch one)
> The AddOn is still then running (as seen in your Dashboard) and in my case there is no Zigbee2MQTT icon in front of it.
> (3) Stop the existing AddOn, remove it from side bar if it was
> [ continue as davidstosik tested approach ]
> (4) install new repository and install correspondig AddOn
> (5) configure it by pasting back your previous AddOn config
> Z2M AddOn would then use all files left by previous AddOn, inc the devices.
> (6) start it, everything should be now fine on latest (1.20) release
> 
> Last step - not tested yet
> (7) delete the old 1.18 AddOn to leave a clean/migrated solution in place.
> 
> That way I did no mess up between new and old because at first they look exactly the same.

Hi,
i had also problems by updating the add-on. I follow the steps till step 6. At starting the add-on I get the error:
"Port '8485' is already in use by something else on the host."

**Comment by ciotlosm** [comment]
Is anyone else still not seeing 1.20? It work for me with proper repo configuration

**Comment by ciotlosm** [comment]
Steps to update found here: https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/issues/52#issuecomment-792873888
